t/batches.t

```diff
@@ -0,0 +1,335 @@
+use strict;
+use warnings;
+use 5.14.2;
+
+use Cwd;
+use Data::Dumper;
+use File::Temp qw[tempdir];
+use JSON::PP;
+use POSIX qw( strftime );
+use Time::Local qw( timelocal_modern );
+use Test::Exception;
+use Test::More;    # see done_testing()
+
+use Zonemaster::Engine;
+
+# Use the TARGET environment variable to set the database to use
```

Comment:
The TARGET environment variable is used for Travis testing. Maybe should be noted here.

Reply:
This variable is not only used by Travis. For instance I use it when running the tests locally.

Reply:
Still, it is assumed by Travis, and that should be noted here.

Reply:
I chose to document this variable in the new EnvironmentVariable.md document, see #993.

Reply:
That is a good choice.